Tweak expected IR for goto labels

llvm-mirror/llvm@6df50f4718cb47dcd669eabe425917263e4a10f1 introduced label syntax that broke a CodeGen test -- `; <label>:0:` became `0:`

Full `check-fort` output is below:

```
[1/2] Running Fort regression tests
llvm-lit: /usr/home/petr/buildbot/fort-worker/fort-intree/llvm/tools/fort/test/lit.cfg:135: note: using fort: '/usr/home/petr/buildbot/fort-worker/fort-intree/build/bin/fort'
-- Testing: 193 tests, 4 threads --
Testing: 0 .. 10
FAIL: Fort :: CodeGen/goto.f95 (27 of 193)
******************** TEST 'Fort :: CodeGen/goto.f95' FAILED ********************
Script:
--
: 'RUN: at line 1';    /usr/home/petr/buildbot/fort-worker/fort-intree/build/bin/fort  -emit-llvm -o - /usr/home/petr/buildbot/fort-worker/fort-intree/llvm/tools/fort/test/CodeGen/goto.f95 | /usr/home/petr/buildbot/fort-worker/fort-intree/build/bin/FileCheck /usr/home/petr/buildbot/fort-worker/fort-intree/llvm/tools/fort/test/CodeGen/goto.f95
--
Exit Code: 1
Command Output (stderr):
--
/usr/home/petr/buildbot/fort-worker/fort-intree/llvm/tools/fort/test/CodeGen/goto.f95:4:24: error: CHECK: expected string not found in input
1000 CONTINUE ! CHECK: ; <label>:0
                       ^
<stdin>:1:1: note: scanning from here
; ModuleID = '/usr/home/petr/buildbot/fort-worker/fort-intree/llvm/tools/fort/test/CodeGen/goto.f95'
^
<stdin>:12:3: note: possible intended match here
 br label %0
  ^
--
********************
Testing: 0 .. 10.. 20.. 30.. 40.. 50.. 60.. 70.. 80.. 90.. 
Testing Time: 11.69s
********************
Failing Tests (1):
    Fort :: CodeGen/goto.f95
  Expected Passes    : 192
  Unexpected Failures: 1
FAILED: tools/fort/test/CMakeFiles/check-fort 
cd /usr/home/petr/buildbot/fort-worker/fort-intree/build/tools/fort/test && /usr/home/petr/buildbot/sandbox/bin/python2.7 /usr/home/petr/buildbot/fort-worker/fort-intree/build/./bin/llvm-lit -sv --param fort_site_config=/usr/home/petr/buildbot/fort-worker/fort-intree/build/tools/fort/test/lit.site.cfg /usr/home/petr/buildbot/fort-worker/fort-intree/build/tools/fort/test
ninja: build stopped: subcommand failed.
program finished with exit code 1
elapsedTime=15.453015
```

--- a/test/CodeGen/goto.f95
+++ b/test/CodeGen/goto.f95
@@ -1,7 +1,7 @@
 ! RUN: %fort -emit-llvm -o - %s | %file_check %s
 PROGRAM gototest
 
-1000 CONTINUE   ! CHECK: ; <label>:0
+1000 CONTINUE   ! CHECK: 0:
      GOTO 1000  ! CHECK: br label %0
 
      GOTO 2000

